scripts/benchmarks/merge-benchmark-data.js

```diff
@@ -0,0 +1,43 @@
+const BENCHMARK_LAYERS = [
+  '../../staging/benchmarks/benchmarks_base_2017.json',
+  '../../staging/benchmarks/benchmarks_updates_20171109.json',
+  '../../staging/benchmarks/benchmarks_readmission_20171109.json'
+];
+
+const UNIQUE_COLUMN_CONSTRAINT = [
+  'measureId',
+  'benchmarkYear',
+  'performanceYear',
+  'submissionMethod'
+];
+
+const getBenchmarkKey = (benchmark) => {
+  let benchmarkKey = '';
+  UNIQUE_COLUMN_CONSTRAINT.forEach((keyName) => {
+    benchmarkKey += `${benchmark[keyName]}|`;
+  });
+
+  return benchmarkKey;
+};
+
+const mergeBenchmarkLayers = (benchmarkLayers) => {
+  const mergedBenchmarks = new Map();
+
+  benchmarkLayers.forEach((benchmarkLayer) => {
+    const benchmarkFile = require(benchmarkLayer);
```

Comment:
let's avoid requiring inside function calls and put requires at the top of the file; while we're technically allowed to lazily require, it's good practice to follow es6 module rules for import/export, as we'll eventually switch to that module format.

Reply:
yea, it seemed sloppy to me too. How does `    const benchmarkFile = JSON.parse(fs.readFileSync(path.join(__dirname, benchmarkLayer), 'utf8'));
` look?

Reply:
borrowed it from the `enrich-measures-data.js` file

Reply:
hm, works for me!